yarn.lock

```diff
@@ -100,6 +100,18 @@
   version "0.2.0"
   resolved "https://registry.yarnpkg.com/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.0.tgz#6bcf6481030f59e7ffd1a40a67cd464d47780f6a"
 
+"@fortawesome/free-brands-svg-icons@^5.1.0":
```

Comment:
Is this still needed? Didn't see a change in package.json so wasn't sure where these came from.

Reply:
Yeah the whole yarn.lock is not needed I wanted to add all FA icons not only solid so that's how it got there.